components/card/index.css

```diff
@@ -85,88 +173,106 @@ governing permissions and limitations under the License.
-  border-top-right-radius: calc(var(--spectrum-card-border-radius) - 1px);
-  border-bottom-right-radius: 0;
-  border-bottom-left-radius: 0;
-}
-
-.spectrum-Card-header {
-  block-size: var(--spectrum-card-body-header-height);
+  border-start-start-radius: var(--mod-card-corner-radius, var(--spectrum-card-corner-radius));
+  border-start-end-radius: var(--mod-card-corner-radius, var(--spectrum-card-corner-radius));
+  border-end-end-radius: 0;
+  border-end-start-radius: 0;
+  display: flex;
+  align-items: center;
 }
 
 .spectrum-Card-content {
   display: flex;
-  min-block-size: var(--spectrum-card-body-content-min-height);
-  margin-block-start: var(--spectrum-card-body-content-margin-top);
+  align-items: center;
+  margin-block-start: var(--mod-card-content-margin-top, var(--spectrum-card-content-margin-top));
 }
 
 .spectrum-Card-title {
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
-  padding-inline-end: var(--spectrum-card-title-padding-right);
+  padding-inline-end: var(--mod-card-title-padding-right, var(--spectrum-card-title-padding-right));
+  font-family: var(--mod-card-title-font-family, var(--spectrum-card-title-font-family));
+  font-size: var(--mod-card-title-font-size, var(--spectrum-card-title-font-size));
+  font-weight: var(--mod-card-title-font-weight, var(--spectrum-card-title-font-weight));
+  font-style: var(--mod-card-title-font-style, var(--spectrum-card-title-font-style));
+  line-height: var(--mod-card-title-line-height, var(--spectrum-card-title-line-height));
+  color: var(--highcontrast-card-title-font-color, var(--mod-card-title-font-color, var(--spectrum-card-title-color)));
 }
 
-.spectrum-Card-subtitle {
-  padding-inline-end: var(--spectrum-card-subtitle-padding-right);
-}
 
-.spectrum-Card-description {
-  font-size: var(--spectrum-card-subtitle-text-size);
+.spectrum-Card-subtitle {
+  padding-inline-end: var(--mod-card-subtitle-padding-right, var(--spectrum-card-subtitle-padding-right));
+  color: var(--highcontrast-card-title-color, var(--mod-card-title-color, var(--spectrum-card-title-color)));
+  text-transform: none;
 }
 
 .spectrum-Card-subtitle + .spectrum-Card-description:before {
   content: "•";
-  padding-inline-end: var(--spectrum-card-subtitle-padding-right);
+  padding-inline-end: var(--mod-card-subtitle-padding-right, var(--spectrum-card-subtitle-padding-right));
+}
+
+.spectrum-Card-description {
+  font-family: var(--mod-card-body-font-family, var(--spectrum-card-body-font-family));
+  font-size: var(--mod-card-body-font-size, var(--spectrum-card-body-font-size));
+  font-weight: var(--mod-card-body-font-weight, var(--spectrum-card-body-font-weight));
+  font-style: var(--mod-card-body-font-style, var(--spectrum-card-body-font-style));
+  line-height: var(--mod-card-body-line-height, var(--spectrum-card-body-line-height));
+  color: var(--highcontrast-card-body-font-color, var(--mod-card-body-font-color, var(--spectrum-card-body-font-color)));
```

Comment:
Do we also need to provide `.spectrum-Card-footer` with the same `color` definition? I ask, because when I switch to dark, the footer text is still coming through with its default color:

<img width="302" alt="Screen Shot 2023-04-17 at 4 47 55 PM" src="https://user-images.githubusercontent.com/360251/232607429-daa97dff-0c73-4674-9c11-5cc007a9b991.png">

Reply:
Yes I'll do that! Footer did not have any specific tokens so I was doing my best here to keep this backwards compatible so we'll definitely want some color